configure

```diff
@@ -1989,6 +1993,25 @@ Available commands are:
       message.index("\n") != nil
     end
   end
+
+  # Returns true if MacPorts is installed in its default location.
+  def macports?
+    File.exists? '/opt/local/bin/port'
+  end
+
+  # Query MacPorts for the path to the latest installed version of
+  # llvm-config that is within the range of supported LLVM versions.
+  def macports_llvm_config
+    supported_versions = (3.0 .. 3.5)
+    installed_ports    = %x(port installed | egrep -o 'llvm-[^ ]+' ).split
```

Comment:
@jc00ke I have no objection to converting the style to backticks. I'll update the pull request later today.